storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -4214,6 +4203,31 @@ private void maybeAddPlaceholderBoltAggStats(TopologyPageInfo topoPageInfo, Stor
         }
     }
 
+    private CommonAggregateStats getPlaceholderCommonAggregateStats(Object component) {
+        // common aggregate
+        CommonAggregateStats commonStats = new CommonAggregateStats();
+
+        // get num_executors
+        int numExecutors = 0;
+        try {
+            numExecutors = "(component);
```

Comment:
This code becomes odd. Please make sure you've run tests locally before push.

Reply:
Sorry, not sure how this happened. I ran tests locally before committing and then must have fat fingered the file before committing.

Reply:
Fix has been committed.